Zend/zend_compile.c

```diff
@@ -6138,35 +6140,35 @@ void zend_begin_method_decl(zend_op_array *op_array, zend_string *name, zend_boo
 	} else if (zend_string_equals_literal(lcname, ZEND_CLONE_FUNC_NAME)) {
 		ce->clone = (zend_function *) op_array;
 	} else if (zend_string_equals_literal(lcname, ZEND_CALL_FUNC_NAME)) {
-		zend_check_magic_method_attr(fn_flags, "__call", 0);
+		zend_check_magic_method_attr(fn_flags, ce, ZEND_CALL_FUNC_NAME, 0);
```

Comment:
This is debatable, but I'd prefer to always keep the `"__call"` form here, rather than sometimes using the constant or not, depending on whether the method name contains uppercase characters.

Reply:
I'll remove those changes then, no problem :)